src/test/java/org/sagebionetworks/bridge/exporter/integration/ExportTest.java

```diff
@@ -0,0 +1,1142 @@
+                tableContents = queryResultAppVersion.getQueryResult().getQueryResults().getRows();
+                if (tableContents.size() != 0) {
+                    break;
+                }
+
+                TimeUnit.SECONDS.sleep(SYNAPSE_RETRY_DELAY);
+
+                jobIdTokenAppVersion = synapseClient.queryTableEntityBundleAsyncStart("select * from " + appVersionTableId
+                                + " where recordId=\'" + uploadStatus.getRecord().getId() + "\'"
+                        , 0L, 100L, true, 0xF, appVersionTableId);
+            }
+
+            // verify the size of the table is equal to the queried upload
+            assertEquals(1, tableContents.size());
+
+            List<SelectColumn> tableHeaders = queryResultAppVersion.getQueryResult().getQueryResults().getHeaders();
+            List<String> rowContent = tableContents.get(0).getValues();
+
+            // verify each column
+            for (int i = 0; i < tableHeaders.size(); i++) {
+                String headerName = tableHeaders.get(i).getName();
+                String columnValue = rowContent.get(i);
+
+                switch (headerName) {
+                    case "recordId": {
+                        assertEquals(uploadStatus.getRecord().getId(), columnValue);
+                        break;
+                    }
+                    case "appVersion": {
+                        assertEquals("version 1.0.0, build 1", columnValue);
+                        break;
+                    }
+                    case "phoneInfo": {
+                        assertEquals("Integration Tests", columnValue);
+                        break;
+                    }
+                    case "uploadDate": {
+                        assertEquals(uploadRecord.getString("uploadDate"), columnValue);
+                        break;
+                    }
+                    case "healthCode": {
+                        assertEquals(uploadRecord.getString("healthCode"), columnValue);
+                        break;
+                    }
+                    case "externalId": {
+                        assertTrue(StringUtils.isBlank(columnValue));
+                        break;
+                    }
+                    case "dataGroups": {
+                        assertTrue(StringUtils.isBlank(columnValue));
+                        break;
+                    }
+                    case "createdOn": {
+                        assertEquals(String.valueOf(dateTimeNow.getMillis()), columnValue);
+                        break;
+                    }
+                    case "createdOnTimeZone": {
+                        ZonedDateTime zDateTime = ZonedDateTime.parse(dateTimeNow.toString(), DateTimeFormatter.ISO_ZONED_DATE_TIME);
+                        String zoneStr = zDateTime.getZone().getId().replaceAll(":", "");
+                        assertEquals(zoneStr, columnValue);
+                        break;
```

Comment:
I'm pretty sure you control the time zone that gets passed into Bridge. So instead of calculating, you can explicitly set the timezone in the test, and then just test against a simple string.